src/redux/redux-reducers/favorites.jsx

```diff
@@ -0,0 +1,13 @@
+import {createSlice} from '@reduxjs/toolkit';
+//reducer for favorite movies
+const favorites = createSlice({
+  name: 'favorite',
+  initialState: [],
+  reducers: {
+    addFavMovies(state, action) {2
```

Comment:
What's this number 2, and what we are returning.
Also, let's try too keep names, more meaningful for example addFavorites